cmd/spicedb/servetesting_integration_test.go

```diff
@@ -159,13 +160,14 @@ func newTester(t *testing.T, containerOpts *dockertest.RunOptions) (*spicedbHand
 		}
 	}
 
+	conn, err := grpc.Dial(fmt.Sprintf("localhost:%s", port),
```

Comment:
Did you move this to a separate PR or something?

Reply:
I just removed it since it was unrelated. I'll have to touch this file when I update the test server anyway (so the linter will complain that I'm using a deprecated api and I'll have to change it there)